### PR TITLE
gh actions caching

### PR DESCRIPTION
**Pull request by BaderBC**

### CODE REVIEW

**Comment by BaderBC** [comment]
If you wonder why this doesn't pass look at #109

**Review by maxidragon — COMMENTED** [review]
I think this won't work. Maybe we can use [docker-cache](https://github.com/marketplace/actions/docker-cache)?

**Comment by BaderBC** [comment]
> I think this won't work. Maybe we can use [docker-cache](https://github.com/marketplace/actions/docker-cache)?

I will try this, but fristly @cooligus should resolve his #109

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -13,11 +13,20 @@ jobs:
...
+        run: docker pull archlinux
+
+      - name: docker pull artixlinux/openrc
+        run: docker pull artixlinux/openrc
+
+      - name: docker pull artixlinux/runit
+        run: docker pull artixlinux/runit
+
+      - uses: actions/checkout@v4
+
```
Comment (cooligus) [inline .github/workflows/tests.yml:29]: Why do you pull these images (9 times)?

**Review by cooligus — CHANGES_REQUESTED** [review]

**Comment by BaderBC** [comment]
It seems that paradoxically caching docker images and/or layers even slows down tests CI
That's why I close this pr

**Pull request closed without merging**